tests/generate_fixtures/generate_target_anim_files.py

```diff
@@ -23,9 +23,14 @@
 # THE SOFTWARE.
 """Generate target files for pyani-plus ANIm tests.
 
-This script can be run with ``./generate_target_anim_files.py`` in the script's
-directory, or from the project root directory via ``make fixtures``. It will
-regenerate and potentially modify test input files under the fixtures directory.
+This script can be run with
+``python generate_target_anim_files.py \
+    <path_to_inputs_dir> \
+    <path_to_delta_output_dir> \
+    <path_to_filter_output_dir>``
```

Comment:
There is only one output directory (now).

Reply:
Oh! Silly me, the code works. I just need to update the docstring!